Remove subdirs that are non-exisitng in the tarball
or else autoreconf breaks.

--- audacity-minsrc-2.2.2-rc1/lib-src/Makefile.am
+++ audacity-minsrc-2.2.2-rc1/lib-src/Makefile.am
@@ -107,18 +107,6 @@
 SUBDIRS = FileDialog
 OPTIONAL_SUBDIRS =
 
-if USE_LOCAL_EXPAT
-OPTIONAL_SUBDIRS += expat
-endif
-
-if USE_LOCAL_LIBID3TAG
-OPTIONAL_SUBDIRS += libid3tag
-endif
-
-if USE_LOCAL_LIBMAD
-OPTIONAL_SUBDIRS += libmad
-endif
-
 if USE_LOCAL_LIBNYQUIST
 SUBDIRS += libnyquist
 endif
@@ -131,23 +119,10 @@
 OPTIONAL_SUBDIRS += libsoxr
 endif
 
-if USE_LOCAL_LIBSOUNDTOUCH
-OPTIONAL_SUBDIRS += soundtouch
-endif
-
 if USE_LOCAL_LIBTWOLAME
 OPTIONAL_SUBDIRS += twolame
 endif
 
-if USE_LOCAL_LIBVORBIS
-OPTIONAL_SUBDIRS += libogg libvorbis
-endif
-
-if USE_LOCAL_LIBFLAC
-# Note: libflac needs to be linked against (the local or system) libogg.
-OPTIONAL_SUBDIRS += libflac
-endif
-
 if USE_LOCAL_LV2
 SUBDIRS += lv2
 endif
